src/sentry/models/dashboard_widget.py

```diff
@@ -76,6 +77,31 @@ class DashboardWidgetTypes(TypesClass):
     TYPE_NAMES = [t[1] for t in TYPES]
 
 
+class DatasetSourcesTypes(Enum):
```

Comment:
I had to copy this from Discover Saved Queries because if I tried to import it, I was getting circular import errors.